content/blog/cc-product-roadmap-1/index.md

```diff
@@ -0,0 +1,72 @@
+
+### Release 1: User profiles
+At the moment mentees can't really do anything in the plaform, but it would be great if any user can update their profile so mentors can see their goals and expectations when a mentee reach out to them.
+
+For this firt release, we are going to create a new private area for members, the profile page will look like this, from here members can update their profile.
+
+![User profile](./release-1.jpg)
+
+We will allow members to update their avatar as well. For now there will be only two options in the main navigation menu, `Home` and `Mentors`.
+
+The Home link will redirect the user to the profile, later on we want to show several widgets here to provider mentors and mentees with useful information about their mentorships.
+
+The Mentors link will send users to the current app we have, from there people will be able to search for mentors.
+
+### Release 2: Mentor availability and reports
+On top of the user profile, we are going to build the mentor profile page, the design is the same but there will be more fields, as mentors can define communication channels and availability.
+
+We are also planning to build an easy way to report any issues in the mentorships by creating a new reports module. Mentors and mentees will be able to report missconducts and reach out to Coding Coach stuff.
+
+![Mentor profile](./release-2.png)
+
+### Release 3: Apply for mentorships
+One of the problems we currently have is that mentorships are happening outside the platform, a mentee will reach out to a mentor and ask for help, but then we don't know if that relationship is successfull or not.
+
+This is the largest release in the roadmap, because we need to introduce a new process to make the mentorships happen in the platform, this way we are preparing the platform for new features we want to include later on.
+
+In the new process a mentee will apply for a mentorship, by setting their goals and expectaions. All the applications will appear to the mentor in the new mentorships module. 
+
+From there the mentor will be able to review the goals and decide if this is a good fit for both parties, then the mentor will be able to approve or reject the application.
+
+Finally if the mentorship gets accepted, the mentee will be able to see the communication channels as well as some other details on the mentorship.
+
+![Mentorships module](./release-3.jpg)
+
+We have more ideas and plans to help mentors and mentees to succeed in the relationship and we believe this is the fundation we need to build everything on top of this.
+
+### Release 4: Notes and finishing a mentorship
+As a mentor myself I like to keep track of my mentees by keeping some notes, this is very useful specially at the beginning. We will work on introducing a feature to keep notes directly on the mentorhip, this way it will be easier for a mentor to keep everything in a single place.
```

Comment:
```suggestion
As a mentor myself I like to keep track of my mentees by keeping some notes, this is very useful especially at the beginning. We will work on introducing a feature to keep notes directly on the mentorship, this way it will be easier for a mentor to keep everything in a single place.
```